test/fuzz/readMessageBinary.js

```diff
@@ -0,0 +1,22 @@
+import { readMessage } from 'openpgp';
+
+const expected = ['This message / key probably does not conform to a valid OpenPGP format'];
+
+function ignoredError(error) {
+  return expected.some(message => error.message.includes(message));
+}
+
+/**
+ * @param { Buffer } inputData
+ */
+export function fuzz (inputData) {
+  const binaryMessage = `-----BEGIN PGP MESSAGE-----\n\n${inputData.toString('base64')}\n-----END PGP MESSAGE-----`;
+  const bufferedMessage = new Uint8Array(Buffer.from(binaryMessage, 'utf8'));
+  return readMessage({ binaryMessage: bufferedMessage })
+    .catch(error => {
+      if (error.message && !ignoredError(error)) {
```

Comment:
*Question*: This ignored error occurs each time because of this check:
`(peekedBytes[0] & 0x80) === 0)`
here https://github.com/openpgpjs/openpgpjs/blob/v6/src/packet/packet.js#L124

How can I avoid this?

Reply:
This error means that the bytes are not a valid OpenPGP message, and should be rejected. So - for many byte arrays this error will be correct, and should indeed be ignored. In fact, I think there will be many other error messages that are (usually/hopefully) expected and correct, e.g. for malformed packets, etc.

I'm not super sure what's the easiest way to detect whether an error is "expected" or not; certainly things like TypeErrors will be unexpected, but those are also probably unlikely to happen purely based on the message contents (but perhaps not impossible).

Another option could be to check whether the message is "supposed" to be valid, e.g. with another OpenPGP implementation, though that would probably be a lot of work, and you might then run into many minor differences in parsing that may or may not be important (but might still be interesting).